### ISSUE TITLE
CloudFare

### ISSUE THREAD

**Issue by sferez** [DavideViolante/socialblade-com-api#97]
Hi!

I recently since a few days running into CloudFare issue? Do you experience the same on your side or is it just an IP issue on my side that is being flagged?

**Comment by DavideViolante** [comment]
Yes, it seems now puppeteer is blocked. Right now I don't know how to fix, I will look into it when I've time. PR are welcome.
![Screenshot from 2024-05-02 09-18-59](https://github.com/DavideViolante/socialblade-com-api/assets/6844116/b7262ea8-2596-4af0-96e2-b1408324b486)

**Comment by sferez** [comment]
FYI it seems that only the /monthly endpoint is affected by cloudfare. I run it correctly without going under "Detailed Statistics"
